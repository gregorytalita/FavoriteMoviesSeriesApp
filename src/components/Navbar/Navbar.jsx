import React from 'react'
import { Typography, makeStyles, Button } from '@material-ui/core'

const useStyles = makeStyles(theme => ({
  navbar: {
    boxShadow: 'rgba(0, 0, 0, .16) 0px 6px 6px',
    width: '100%',
    height: 80,
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    position: 'relative'
  },
  dashboard: {
    background: 'linear-gradient(to bottom, #F48087, #E75B64)',
    height: 20,
    padding: '16px 40px',
    color: '#fff',
    display: 'flex',
    justifyContent: 'center',
    borderRadius: 68
  },
  logout: {
    position: 'absolute',
    right: 32
  }
}))

const Navbar = ({ logout }) => {
  const classes = useStyles()

  return (
    <div className={classes.navbar}>
      <div className={classes.dashboard}>
        <Typography>
          Dashboard
        </Typography>
      </div>
      <Button color='secondary' className={classes.logout} onClick={logout}>
        Logout
      </Button>
    </div>
  )
}

export default Navbar
