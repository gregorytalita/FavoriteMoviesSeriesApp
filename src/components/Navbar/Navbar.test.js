import React from 'react'
import { create } from 'react-test-renderer'
import Navbar from './Navbar'

describe('Navbar component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Navbar />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
