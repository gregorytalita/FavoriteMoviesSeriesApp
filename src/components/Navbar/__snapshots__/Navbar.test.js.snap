// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component it matches the snapshot 1`] = `
<div
  className="makeStyles-navbar-1"
>
  <div
    className="makeStyles-dashboard-2"
  >
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      Dashboard
    </p>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root makeStyles-logout-3 MuiButton-text MuiButton-textSecondary"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Logout
    </span>
  </button>
</div>
`;
